tests/python/mock_jira.py

```diff
@@ -712,17 +730,22 @@ def search_assignable_users_for_issues(self, name, startAt=0, maxResults=20, *ar
             # Mock Jira REST api bug
             return []
 
-        if startAt == 0:
-            return [User(None, None, JIRA_USER_2)] * maxResults
-        return [User(None, None, JIRA_USER)]
+        # Create a list of users.
+        users = [User(None, None, JIRA_USER_2)] * maxResults + [User(None, None, JIRA_USER)]
 
-    def user(self, id):
+        # Return the requested slice.
+        return users[startAt: startAt + maxResults]
+
+    def user(self, id, payload="username"):
         """
         Mocked Jira method.
         Return :class:`JiraUser`.
         """
-        if id == JIRA_USER["key"]:
+        if payload == "username":
+            payload = "name"
```

Comment:
Why's the default "username" if you're just going to turn it into "name"?

Reply:
Excellent question! So, if you remember this is the mock implementation of the `user` method I modified on the JIRA fork. So I wanted to replicate the signature of the method. I'll add a couple of comments to the code for clarifying the intent.

Here I am modifying